demo/lib/host.bash

```diff
@@ -150,6 +150,7 @@ host-create-vm() {
     else
         echo "Warning: cannot verify Qemu version on govm image. In case of failure, check it is >= 5.0" >&2
     fi
+    echo "# VM Qemu monitor: docker exec -it $VM_CONTAINER_ID nc local:/data/monitor"
```

Comment:
This is a way to connect to Qemu monitor without sudo in order to access `-monitor unix:socket-file` or change network namespace to connect to `-monitor tcp::port`.